### PR TITLE
Added with_timezone flag for Yahoo Placefinder

### PR DESCRIPTION
**Pull request by SoftwareArtisan**
Should have a more generic approach to adding BOSS API flags.  Chose only to implement timezone.

### CODE REVIEW

File: geopy/geocoders/placefinder.py [hunk 1]
```diff
@@ -163,6 +165,8 @@ def geocode(
             params["gflags"] = "R"
         if exactly_one is True:
             params["count"] = "1"
+        if with_timezone is True:
+            params['flags'] += 'T' #Return timezone
```
Comment (ijl) [inline geopy/geocoders/placefinder.py]: `+=` is a typo, yeah?

File: test/geocoders/placefinder.py [hunk 2]
```diff
@@ -55,3 +55,12 @@ def test_reverse_point(self):
...
             {"latitude": 40.75376406311989, "longitude": -73.98489005863667}
         )
+
+    def test_timezone(self):
+        """
+        YahooPlacefinder.with_timezone
+        """
+        self.geocode_run(
+            {"query": "nyc", 'with_timezone':True},
+            {"latitude": 40.71455, "longitude": -74.00712},
```
Comment (ijl) [inline test/geocoders/placefinder.py]: Do you expect different data in the `Location.raw` property?

**Comment by ijl** [comment]
@SoftwareArtisan okay, thanks, two minor questions.

File: geopy/geocoders/placefinder.py (hunk 1 above)
Comment (SoftwareArtisan) [inline geopy/geocoders/placefinder.py]: This works fine - adds a 'T'

File: test/geocoders/placefinder.py (hunk 2 above)
Comment (SoftwareArtisan) [inline test/geocoders/placefinder.py]: Location.raw will now have a timezone entry

File: geopy/geocoders/placefinder.py (hunk 1 above)
Comment (ijl) [inline geopy/geocoders/placefinder.py]: Misread that, sorry.

**Comment by ijl** [comment]
Merged in 502022821c4637e5d16ff85ba94e84e9a9de635a, thanks.

**Comment by ijl** [comment]
Released in 1.10.0.

**Pull request closed without merging**